src/main/resources/messages/messages_nl.yml

```diff
@@ -119,18 +120,19 @@ captcha:
     usage_captcha: '&3Om in te loggen moet je een captcha-code oplossen, gebruik het commando: /captcha %captcha_code'
     wrong_captcha: '&cVerkeerde captcha-code, typ alsjeblieft "/captcha %captcha_code" in de chat!'
     valid_captcha: '&2De captcha-code is geldig!'
-    # TODO captcha_for_registration: 'To register you have to solve a captcha first, please use the command: /captcha %captcha_code'
-    # TODO register_captcha_valid: '&2Valid captcha! You may now register with /register'
+    # TODO (Translator): Moeten wij het woord 'captcha' laten staan? Of is puzzel een betere verwoording?
+    captcha_for_registration: 'Om je te registreren moet je eerst een puzzel oplossen. Gebruik alsjeblieft het volgende commando: /captcha %captcha_code'
+    register_captcha_valid: 'Valide puzzel! Je mag je nu registreren met /register'
```

Comment:
Again, not a native speaker, but if you do see "captcha" in NL-language sites I would vote for that as it's more specific than _puzzel_. I see Wikipedia uses (captcha)[https://nl.wikipedia.org/wiki/Captcha]. But consistency is more important, IMHO—you can choose which one is more appropriate and we can look that all the messages will use the same terminology.

Reply:
I changed it to captcha. I could only find one reference to captcha. Maybe this should be revisited later but for now it's the best i can make of it.